salt/_macros/certs.jinja

```diff
@@ -88,6 +88,7 @@
     - ST: {{ pillar['certificate_information']['subject_properties']['ST']|yaml_dquote }}
     - basicConstraints: "critical CA:false"
     - keyUsage: nonRepudiation, digitalSignature, keyEncipherment
+    - extendedKeyUsage: serverAuth, clientAuth
```

Comment:
This is making every cert a client cert, why are we doing this?

Reply:
Yeah, I will remove this just in case...